armory/data/datasets.py

```diff
@@ -1,5 +1,10 @@
 """
 Download and preprocess common datasets.
+Each standard dataset resides in its own subdirectory under dataset_dir based
+upon the name of the function in the datasets file. For example, the cifar10
+data is found at 'dataset_dir/cifar10'
+The 'download' subdirectory under dataset_dir is reserved for caching.
+The 'private' subdirectory under dataset_dir is reserved for private datasets.
```

Comment:
Sorry, one more! (I think that this is the last one.) "dataset_dir" is not an actual directory name, but a placeholder for a key in the armory config. I would replace every instance of dataset_dir with <dataset_dir> to make it clear that it's not a string literal.